hicexplorer/hicAdjustMatrix.py

```diff
@@ -111,9 +111,7 @@ def adjustMatrix(pArgs):
         # log.debug('genomic_regions {}'.format(genomic_regions))
         matrix_indices_regions = []
         for region in genomic_regions:
-            _regionBinRange = hic_matrix.getRegionBinRange(region[0],
-                                                           int(region[1]),
-                                                           int(region[2]))
+            _regionBinRange = hic_matrix.getRegionBinRange(region[0], int(region[1]), int(region[2]))
```

Comment:
That's all? There were more changes in the fix_polarization PR?!

Reply:
tests are also added. You have also asked to change some variable names which have changed in another PR and it is already merged.